res/views/layout.html.twig

```diff
@@ -87,7 +87,7 @@
         <script src="/dist/js/main.min.js?v={{ version }}"></script>
 
         <script src="https://cdn.jsdelivr.net/algoliasearch/3/algoliasearch.min.js"></script>
-        <script src="https://cdn.jsdelivr.net/instantsearch.js/1/instantsearch.min.js"></script>
+        <script src="https://cdn.jsdelivr.net/instantsearch.js/1/instantsearch.min.js" async></script>
```

Comment:
this is the very last file in the algolia-request chain to be loaded, therefore can be loaded asynchronously (after algoliasearch.min successfully loaded synchronously)

Reply:
That makes sense indeed, can I merge?

Reply:
I guess the patch is complete, but atm untested